core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -914,11 +915,10 @@ public void activateLaunchedExperimentDoesNotDispatchEvent() throws Exception {
         // Force variation to launched experiment.
         optimizely.setForcedVariation(launchedExperiment.getKey(), testUserId, expectedVariation.getKey());
 
-        logbackVerifier.expectMessage(Level.INFO,
-            "Experiment has \"Launched\" status so not dispatching event during activation.");
         Variation variation = optimizely.activate(launchedExperiment.getKey(), testUserId);
         assertNotNull(variation);
         assertThat(variation.getKey(), is(expectedVariation.getKey()));
+        eventHandler.expectImpression(launchedExperiment.getId(), expectedVariation.getId(), testUserId);
```

Comment:
Not sure if I miss it, but I do not see validation of new metadata in expected events.